mllib/src/main/scala/org/apache/spark/ml/tuning/CrossValidator.scala

```diff
@@ -191,7 +194,8 @@ object CrossValidator extends MLReadable[CrossValidator] {
 
 /**
  * :: Experimental ::
- * Model from k-fold cross validation.
+ * CrossValidatorModel contains the model that achieved the highest average cross-validation
```

Comment:
I think we could maybe make this a bit clearer. Maybe something like:

"CrossValidatorModel contains the model with the highest average cross validation metric across folds and uses this model to transform input data. CrossValidatorModel also tracks the metrics for each param map evaluated."

This way its clear that it contains both the best model and the metrics, as well as uses the best model if asked to predict/transform any data. What do you think?
(Of course if we change it here we would want to have the corresponding change occur in Python).